### PR TITLE
extract formatter from cell context

### PR DESCRIPTION
**Pull request by itkg-nanne**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2153
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/359

### CODE REVIEW

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/SearchFormGroup/DateForm.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import TemplateManager         from '../../TemplateManager'
+import TemplateManager         from '../TemplateManager'
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/SearchFormGroup/DateForm.js]: ??

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/CellFormatter/TextFormatter.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import AbstractCellFormatter from './AbstractCellFormatter'
+import TextDataFormatter     from '../DataFormatter/TextFormatter'
+
+/**
+ * @class TextFormatter
+ */
+class TextFormatter extends AbstractCellFormatter
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/CellFormatter/TextFormatter.js]: why this class i dont understand. Why not use directly DataFormatter/TextFormatter

**Comment by itkg-nanne** [comment]
update done @alavieille

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/SearchFormGroup/TextForm.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import TemplateManager         from '../../TemplateManager'
+import TemplateManager         from '../TemplateManager'
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/SearchFormGroup/TextForm.js]: can you move also template outside content ?

**Comment by ngilain** [comment]
:+1:

**Pull request merged**